server/Arcadia.Assistant.CSP/Vacations/ArcadiaVacationRegistry.cs

```diff
@@ -2,27 +2,42 @@
 {
     using System;
     using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
 
     using Akka.Actor;
     using Akka.Event;
 
+    using Arcadia.Assistant.Calendar.Abstractions;
+    using Arcadia.Assistant.Calendar.Abstractions.EventBus;
     using Arcadia.Assistant.Configuration.Configuration;
+    using Arcadia.Assistant.CSP.Model;
     using Arcadia.Assistant.Organization.Abstractions;
 
     public class ArcadiaVacationRegistry : VacationsRegistry, ILogReceive
     {
         private readonly ILoggingAdapter logger = Context.GetLogger();
 
         private readonly VacationsQueryExecutor vacationsQueryExecutor;
+        private readonly VacationsPersistenceExecutor vacationsPersistenceExecutor;
```

Comment:
LOL. Persistence executor :) Reading is also a part of Persistence. VacationsDbWrite, VacationsDbReader?

Reply:
`DbWriter` is too common name. As we distinguish db services by their functions, it should be like `VacationsSyncExecutor` (if you like this word so much).

Reply:
Renamed.